modules/API/API.js

```diff
@@ -380,6 +381,9 @@ function initCommands() {
             sendAnalytics(createApiEvent('screen.sharing.toggled'));
             toggleScreenSharing(options.enable);
         },
+        'toggle-noise-suppression': () => {
```

Comment:
Please add an enabled parameter. Otherwise the user needs to keep track of the existing state, which kinda sucks. I know we have existing similar code, but we have been gradually going for this model.

Reply:
Got it, changed it.